bluepysnap/frame_report.py

```diff
@@ -211,6 +212,10 @@ def nodes(self):
 
     def _resolve(self, group):
         """Transform a group into a node_id array."""
+        if group == []:
+            # Only solution to return an empty list from self._frame_population.get
+            # see: https://github.com/BlueBrain/libsonata/issues/84
+            return np.array([-2])
```

Comment:
You could define

```python
def empty_list():
    '''Only solution to return an empty list from self._frame_population.get
       see: https://github.com/BlueBrain/libsonata/issues/84
    '''
   return np.array([-2])
```
And use it everywhere, so that you won't have to look for np.array([-2]) the day you need to fix it, and you don't need to rewrite the comment everytime you need it.

Reply:
yes why not.